App keeps crashing on Android

Not sure what is causing this.  Happens on my phone and in an emulator, both on the latest available versions.

<details>
<summary>Crash logs from emulator</summary>
<pre>
11-10 01:51:57.143  1508  2974 I ActivityManager: START u0 {act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] flg=0x10200000 cmp=com.allaboutolaf/.MainActivity bnds=[437,1207][644,1485]} from uid 10020
11-10 01:51:57.151  1321  3497 W audio_hw_generic: Not supplying enough data to HAL, expected position 2765072 , only wrote 2764800
11-10 01:51:57.183  5114  5114 I zygote  : Not late-enabling -Xcheck:jni (already on)
11-10 01:51:57.187  1508  2221 I ActivityManager: Start proc 5114:com.allaboutolaf/u0a86 for activity com.allaboutolaf/.MainActivity
11-10 01:51:57.191  1508  1522 E memtrack: Couldn't load memtrack module
11-10 01:51:57.191  1508  1522 W android.os.Debug: failed to get memory consumption info: -1
11-10 01:51:57.191  1324  1324 D gralloc_ranchu: gralloc_alloc: Creating ashmem region of size 8298496
11-10 01:51:57.220  5114  5114 W zygote  : Unexpected CPU variant for X86 using defaults: x86
11-10 01:51:57.249  1324  1324 D gralloc_ranchu: gralloc_alloc: Creating ashmem region of size 4096
11-10 01:51:57.327  2056  3565 E ContentStoreEUAS: Failed to commit the deferred actions
11-10 01:51:57.337  2056  3550 D EGL_emulation: eglMakeCurrent: 0xa13051e0: ver 2 0 (tinfo 0xa13033b0)
11-10 01:51:57.351  2103  2217 D EGL_emulation: eglMakeCurrent: 0xaa8218c0: ver 2 0 (tinfo 0xaa86cf90)
11-10 01:51:57.356  5114  5114 V fb-UnpackingSoSource: locked dso store /data/user/0/com.allaboutolaf/lib-main
11-10 01:51:57.358  5114  5114 I fb-UnpackingSoSource: dso store is up-to-date: /data/user/0/com.allaboutolaf/lib-main
11-10 01:51:57.358  5114  5114 V fb-UnpackingSoSource: releasing dso store lock for /data/user/0/com.allaboutolaf/lib-main
11-10 01:51:57.386  1508  1522 E memtrack: Couldn't load memtrack module
11-10 01:51:57.386  1508  1522 W android.os.Debug: failed to get memory consumption info: -1
11-10 01:51:57.419  5114  5114 D NetworkSecurityConfig: No Network Security Config specified, using platform default
11-10 01:51:57.488  5114  5114 I zygote  : Thread[1,tid=5114,Native,Thread*=0xaa85a000,peer=0x72b4a568,"main"] recursive attempt to load library "/data/app/com.allaboutolaf-w2SIV_V0DGS4CaY36VQmFA==/lib/x86/libfb.so"
11-10 01:51:57.513  5114  5114 W unknown:React: Packager connection already open, nooping.
11-10 01:51:57.513  5114  5114 W unknown:React: Inspector connection already open, nooping.
11-10 01:51:57.524  5114  5134 I zygote  : Rejecting re-init on previously-failed class java.lang.Class<com.google.android.gms.maps.GoogleMapOptions>: java.lang.NoClassDefFoundError: Failed resolution of: Lcom/google/android/gms/common/internal/safeparcel/zza;
11-10 01:51:57.524  5114  5134 I zygote  :   at void com.airbnb.android.react.maps.AirMapManager.<init>(com.facebook.react.bridge.ReactApplicationContext) (AirMapManager.java:52)
11-10 01:51:57.524  5114  5134 I zygote  :   at java.util.List com.airbnb.android.react.maps.MapsPackage.createViewManagers(com.facebook.react.bridge.ReactApplicationContext) (MapsPackage.java:39)
11-10 01:51:57.524  5114  5134 I zygote  :   at java.util.List com.facebook.react.ReactInstanceManager.createAllViewManagers(com.facebook.react.bridge.ReactApplicationContext) (ReactInstanceManager.java:655)
11-10 01:51:57.524  5114  5134 I zygote  :   at com.facebook.react.uimanager.UIManagerModule com.facebook.react.CoreModulesPackage.createUIManager(com.facebook.react.bridge.ReactApplicationContext) (CoreModulesPackage.java:227)
11-10 01:51:57.524  5114  5134 I zygote  :   at com.facebook.react.uimanager.UIManagerModule com.facebook.react.CoreModulesPackage.access$200(com.facebook.react.CoreModulesPackage, com.facebook.react.bridge.ReactApplicationContext) (CoreModulesPackage.java:76)
11-10 01:51:57.524  5114  5134 I zygote  :   at com.facebook.react.bridge.NativeModule com.facebook.react.CoreModulesPackage$8.get() (CoreModulesPackage.java:153)
11-10 01:51:57.524  5114  5134 I zygote  :   at java.lang.Object com.facebook.react.CoreModulesPackage$8.get() (CoreModulesPackage.java:150)
11-10 01:51:57.524  5114  5134 I zygote  :   at java.util.List com.facebook.react.LazyReactPackage.createNativeModules(com.facebook.react.bridge.ReactApplicationContext) (LazyReactPackage.java:82)
11-10 01:51:57.524  5114  5134 I zygote  :   at void com.facebook.react.NativeModuleRegistryBuilder.processPackage(com.facebook.react.ReactPackage) (NativeModuleRegistryBuilder.java:106)
11-10 01:51:57.524  5114  5134 I zygote  :   at void com.facebook.react.ReactInstanceManager.processPackage(com.facebook.react.ReactPackage, com.facebook.react.NativeModuleRegistryBuilder, com.facebook.react.bridge.JavaScriptModuleRegistry$Builder) (ReactInstanceManager.java:1009)
11-10 01:51:57.524  5114  5134 I zygote  :   at com.facebook.react.bridge.ReactApplicationContext com.facebook.react.ReactInstanceManager.createReactContext(com.facebook.react.cxxbridge.JavaScriptExecutor, com.facebook.react.cxxbridge.JSBundleLoader) (ReactInstanceManager.java:933)
11-10 01:51:57.524  5114  5134 I zygote  :   at com.facebook.react.bridge.ReactApplicationContext com.facebook.react.ReactInstanceManager.access$600(com.facebook.react.ReactInstanceManager, com.facebook.react.cxxbridge.JavaScriptExecutor, com.facebook.react.cxxbridge.JSBundleLoader) (ReactInstanceManager.java:109)
11-10 01:51:57.524  5114  5134 I zygote  :   at void com.facebook.react.ReactInstanceManager$4.run() (ReactInstanceManager.java:746)
11-10 01:51:57.524  5114  5134 I zygote  :   at void java.lang.Thread.run() (Thread.java:764)
11-10 01:51:57.524  5114  5134 I zygote  : Caused by: java.lang.ClassNotFoundException: Didn't find class "com.google.android.gms.common.internal.safeparcel.zza" on path: DexPathList[[zip file "/data/app/com.allaboutolaf-w2SIV_V0DGS4CaY36VQmFA==/base.apk"],nativeLibraryDirectories=[/data/app/com.allaboutolaf-w2SIV_V0DGS4CaY36VQmFA==/lib/x86, /system/fake-libs, /data/app/com.allaboutolaf-w2SIV_V0DGS4CaY36VQmFA==/base.apk!/lib/x86, /system/lib, /system/vendor/lib]]
11-10 01:51:57.524  5114  5134 I zygote  :   at java.lang.Class dalvik.system.BaseDexClassLoader.findClass(java.lang.String) (BaseDexClassLoader.java:93)
11-10 01:51:57.524  5114  5134 I zygote  :   at java.lang.Class java.lang.ClassLoader.loadClass(java.lang.String, boolean) (ClassLoader.java:379)
11-10 01:51:57.524  5114  5134 I zygote  :   at java.lang.Class java.lang.ClassLoader.loadClass(java.lang.String) (ClassLoader.java:312)
11-10 01:51:57.524  5114  5134 I zygote  :   at void com.airbnb.android.react.maps.AirMapManager.<init>(com.facebook.react.bridge.ReactApplicationContext) (AirMapManager.java:52)
11-10 01:51:57.524  5114  5134 I zygote  :   at java.util.List com.airbnb.android.react.maps.MapsPackage.createViewManagers(com.facebook.react.bridge.ReactApplicationContext) (MapsPackage.java:39)
11-10 01:51:57.524  5114  5134 I zygote  :   at java.util.List com.facebook.react.ReactInstanceManager.createAllViewManagers(com.facebook.react.bridge.ReactApplicationContext) (ReactInstanceManager.java:655)
11-10 01:51:57.524  5114  5134 I zygote  :   at com.facebook.react.uimanager.UIManagerModule com.facebook.react.CoreModulesPackage.createUIManager(com.facebook.react.bridge.ReactApplicationContext) (CoreModulesPackage.java:227)
11-10 01:51:57.524  5114  5134 I zygote  :   at com.facebook.react.uimanager.UIManagerModule com.facebook.react.CoreModulesPackage.access$200(com.facebook.react.CoreModulesPackage, com.facebook.react.bridge.ReactApplicationContext) (CoreModulesPackage.java:76)
11-10 01:51:57.524  5114  5134 I zygote  :   at com.facebook.react.bridge.NativeModule com.facebook.react.CoreModulesPackage$8.get() (CoreModulesPackage.java:153)
11-10 01:51:57.524  5114  5134 I zygote  :   at java.lang.Object com.facebook.react.CoreModulesPackage$8.get() (CoreModulesPackage.java:150)
11-10 01:51:57.524  5114  5134 I zygote  :   at java.util.List com.facebook.react.LazyReactPackage.createNativeModules(com.facebook.react.bridge.ReactApplicationContext) (LazyReactPackage.java:82)
11-10 01:51:57.524  5114  5134 I zygote  :   at void com.facebook.react.NativeModuleRegistryBuilder.processPackage(com.facebook.react.ReactPackage) (NativeModuleRegistryBuilder.java:106)
11-10 01:51:57.524  5114  5134 I zygote  :   at void com.facebook.react.ReactInstanceManager.processPackage(com.facebook.react.ReactPackage, com.facebook.react.NativeModuleRegistryBuilder, com.facebook.react.bridge.JavaScriptModuleRegistry$Builder) (ReactInstanceManager.java:1009)
11-10 01:51:57.525  5114  5134 I zygote  :   at com.facebook.react.bridge.ReactApplicationContext com.facebook.react.ReactInstanceManager.createReactContext(com.facebook.react.cxxbridge.JavaScriptExecutor, com.facebook.react.cxxbridge.JSBundleLoader) (ReactInstanceManager.java:933)
11-10 01:51:57.525  5114  5134 I zygote  :   at com.facebook.react.bridge.ReactApplicationContext com.facebook.react.ReactInstanceManager.access$600(com.facebook.react.ReactInstanceManager, com.facebook.react.cxxbridge.JavaScriptExecutor, com.facebook.react.cxxbridge.JSBundleLoader) (ReactInstanceManager.java:109)
11-10 01:51:57.525  5114  5134 I zygote  :   at void com.facebook.react.ReactInstanceManager$4.run() (ReactInstanceManager.java:746)
11-10 01:51:57.525  5114  5134 I zygote  :   at void java.lang.Thread.run() (Thread.java:764)
11-10 01:51:57.525  5114  5134 I zygote  :
11-10 01:51:57.534  1333  1428 E SurfaceFlinger: ro.sf.lcd_density must be defined as a build property
11-10 01:51:57.537  5114  5136 D OpenGLRenderer: HWUI GL Pipeline
11-10 01:51:57.539  5114  5134 E AndroidRuntime: FATAL EXCEPTION: Thread-2
11-10 01:51:57.539  5114  5134 E AndroidRuntime: Process: com.allaboutolaf, PID: 5114
11-10 01:51:57.539  5114  5134 E AndroidRuntime: java.lang.NoClassDefFoundError: Failed resolution of: Lcom/google/android/gms/maps/GoogleMapOptions;
11-10 01:51:57.539  5114  5134 E AndroidRuntime: 	at com.airbnb.android.react.maps.AirMapManager.<init>(AirMapManager.java:52)
11-10 01:51:57.539  5114  5134 E AndroidRuntime: 	at com.airbnb.android.react.maps.MapsPackage.createViewManagers(MapsPackage.java:39)
11-10 01:51:57.539  5114  5134 E AndroidRuntime: 	at com.facebook.react.ReactInstanceManager.createAllViewManagers(ReactInstanceManager.java:655)
11-10 01:51:57.539  5114  5134 E AndroidRuntime: 	at com.facebook.react.CoreModulesPackage.createUIManager(CoreModulesPackage.java:227)
11-10 01:51:57.539  5114  5134 E AndroidRuntime: 	at com.facebook.react.CoreModulesPackage.access$200(CoreModulesPackage.java:76)
11-10 01:51:57.539  5114  5134 E AndroidRuntime: 	at com.facebook.react.CoreModulesPackage$8.get(CoreModulesPackage.java:153)
11-10 01:51:57.539  5114  5134 E AndroidRuntime: 	at com.facebook.react.CoreModulesPackage$8.get(CoreModulesPackage.java:150)
11-10 01:51:57.539  5114  5134 E AndroidRuntime: 	at com.facebook.react.LazyReactPackage.createNativeModules(LazyReactPackage.java:82)
11-10 01:51:57.539  5114  5134 E AndroidRuntime: 	at com.facebook.react.NativeModuleRegistryBuilder.processPackage(NativeModuleRegistryBuilder.java:106)
11-10 01:51:57.539  5114  5134 E AndroidRuntime: 	at com.facebook.react.ReactInstanceManager.processPackage(ReactInstanceManager.java:1009)
11-10 01:51:57.539  5114  5134 E AndroidRuntime: 	at com.facebook.react.ReactInstanceManager.createReactContext(ReactInstanceManager.java:933)
11-10 01:51:57.539  5114  5134 E AndroidRuntime: 	at com.facebook.react.ReactInstanceManager.access$600(ReactInstanceManager.java:109)
11-10 01:51:57.539  5114  5134 E AndroidRuntime: 	at com.facebook.react.ReactInstanceManager$4.run(ReactInstanceManager.java:746)
11-10 01:51:57.539  5114  5134 E AndroidRuntime: 	at java.lang.Thread.run(Thread.java:764)
11-10 01:51:57.539  5114  5134 E AndroidRuntime: Caused by: java.lang.ClassNotFoundException: com.google.android.gms.maps.GoogleMapOptions
11-10 01:51:57.539  5114  5134 E AndroidRuntime: 	at java.lang.VMClassLoader.findLoadedClass(Native Method)
11-10 01:51:57.539  5114  5134 E AndroidRuntime: 	at java.lang.ClassLoader.findLoadedClass(ClassLoader.java:738)
11-10 01:51:57.539  5114  5134 E AndroidRuntime: 	at java.lang.ClassLoader.loadClass(ClassLoader.java:363)
11-10 01:51:57.539  5114  5134 E AndroidRuntime: 	at java.lang.ClassLoader.loadClass(ClassLoader.java:312)
11-10 01:51:57.539  5114  5134 E AndroidRuntime: 	... 14 more
11-10 01:51:57.539  5114  5134 E AndroidRuntime: Caused by: java.lang.NoClassDefFoundError: Failed resolution of: Lcom/google/android/gms/common/internal/safeparcel/zza;
11-10 01:51:57.539  5114  5134 E AndroidRuntime: 	... 14 more
11-10 01:51:57.539  5114  5134 E AndroidRuntime: Caused by: java.lang.ClassNotFoundException: Didn't find class "com.google.android.gms.common.internal.safeparcel.zza" on path: DexPathList[[zip file "/data/app/com.allaboutolaf-w2SIV_V0DGS4CaY36VQmFA==/base.apk"],nativeLibraryDirectories=[/data/app/com.allaboutolaf-w2SIV_V0DGS4CaY36VQmFA==/lib/x86, /system/fake-libs, /data/app/com.allaboutolaf-w2SIV_V0DGS4CaY36VQmFA==/base.apk!/lib/x86, /system/lib, /system/vendor/lib]]
11-10 01:51:57.539  5114  5134 E AndroidRuntime: 	at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:93)
11-10 01:51:57.539  5114  5134 E AndroidRuntime: 	at java.lang.ClassLoader.loadClass(ClassLoader.java:379)
11-10 01:51:57.539  5114  5134 E AndroidRuntime: 	at java.lang.ClassLoader.loadClass(ClassLoader.java:312)
11-10 01:51:57.539  5114  5134 E AndroidRuntime: 	... 14 more
11-10 01:51:57.562  1508  1737 W ActivityManager:   Force finishing activity com.allaboutolaf/.MainActivity
11-10 01:51:57.613  5114  5136 D         : HostConnection::get() New Host Connection established 0xa1064e80, tid 5136
11-10 01:51:57.613  1508  1524 I ActivityManager: Showing crash dialog for package com.allaboutolaf u0
11-10 01:51:57.620  1324  1324 D gralloc_ranchu: gralloc_alloc: Creating ashmem region of size 4096
11-10 01:51:57.621  5114  5136 I OpenGLRenderer: Initialized EGL, version 1.4
11-10 01:51:57.621  5114  5136 D OpenGLRenderer: Swap behavior 1
11-10 01:51:57.622  5114  5136 W OpenGLRenderer: Failed to choose config with EGL_SWAP_BEHAVIOR_PRESERVED, retrying without...
11-10 01:51:57.622  5114  5136 D OpenGLRenderer: Swap behavior 0
11-10 01:51:57.631  1324  1324 D gralloc_ranchu: gralloc_alloc: Creating ashmem region of size 4096
11-10 01:51:57.631  5114  5136 D EGL_emulation: eglCreateContext: 0x9e6fd860: maj 2 min 0 rcv 2
11-10 01:51:57.639  1324  1324 D gralloc_ranchu: gralloc_alloc: Creating ashmem region of size 4096
11-10 01:51:57.639  5114  5136 D EGL_emulation: eglMakeCurrent: 0x9e6fd860: ver 2 0 (tinfo 0x9e6a4910)
11-10 01:51:57.725  5114  5136 D         : SurfaceInterface::setAsyncMode: set async mode 1
11-10 01:51:57.737  1324  1324 D gralloc_ranchu: gralloc_alloc: Creating ashmem region of size 4096
11-10 01:51:57.746  1508  2974 I WindowManager: Failed to capture screenshot of Token{f691623 ActivityRecord{18a8352 u0 com.allaboutolaf/.MainActivity t63 f}} appWin=Window{6010a49 u0 com.allaboutolaf/com.allaboutolaf.MainActivity} drawState=1
11-10 01:51:57.789  1508  1524 W Looper  : Dispatch took 179ms on android.ui, h=Handler (com.android.server.am.ActivityManagerService$UiHandler) {c2f2d2f} cb=null msg=1
11-10 01:51:57.783  1324  1324 I chatty  : uid=1000(system) allocator@2.0-s identical 4 lines
11-10 01:51:57.793  1324  1324 D gralloc_ranchu: gralloc_alloc: Creating ashmem region of size 4096
11-10 01:51:57.794  5114  5136 D EGL_emulation: eglMakeCurrent: 0x9e6fd860: ver 2 0 (tinfo 0x9e6a4910)
11-10 01:51:57.805  1324  1324 D gralloc_ranchu: gralloc_alloc: Creating ashmem region of size 4096
11-10 01:51:57.811  1324  1324 D gralloc_ranchu: gralloc_alloc: Creating ashmem region of size 4096
11-10 01:51:57.832  2056  3550 D         : SurfaceInterface::setAsyncMode: set async mode 1
11-10 01:51:57.841  2056  3550 D EGL_emulation: eglMakeCurrent: 0xa13051e0: ver 2 0 (tinfo 0xa13033b0)
11-10 01:51:57.842  2103  2217 D         : SurfaceInterface::setAsyncMode: set async mode 1
11-10 01:51:57.853  1324  1324 D gralloc_ranchu: gralloc_alloc: Creating ashmem region of size 4096
11-10 01:51:57.856  2103  2217 D EGL_emulation: eglMakeCurrent: 0xaa8218c0: ver 2 0 (tinfo 0xaa86cf90)
11-10 01:51:57.869  5114  5136 D EGL_emulation: eglMakeCurrent: 0x9e6fd860: ver 2 0 (tinfo 0x9e6a4910)
11-10 01:51:57.873  1324  1324 D gralloc_ranchu: gralloc_alloc: Creating ashmem region of size 4096
11-10 01:51:57.916  1508  1524 W Looper  : Dispatch took 121ms on android.ui, h=Handler (android.view.Choreographer$FrameHandler) {3ad151} cb=android.view.Choreographer$FrameDisplayEventReceiver@a77fdb6 msg=0
11-10 01:51:57.925  2056  2056 I MicroDetectionWorker: #updateMicroDetector [detectionMode: [mDetectionMode: [1]]]
11-10 01:51:57.925  2056  2056 I MicroDetectionWorker: #startMicroDetector [speakerMode: 0]
11-10 01:51:57.929  2056  2056 I AudioController: Using mInputStreamFactoryBuilder
11-10 01:51:57.935  2056  2056 I MicroDetectionWorker: onReady
11-10 01:51:57.939  2056  2207 I MicroRecognitionRunner: Starting detection.
11-10 01:51:57.941  2056  3565 I MicrophoneInputStream: mic_starting com.google.android.apps.gsa.staticplugins.aa.c@369385
11-10 01:51:57.950  1340  5149 I AudioFlinger: AudioFlinger's thread 0xa7483d80 tid=5149 ready to run
11-10 01:51:57.951  1340  1400 E AudioFlinger: not enough memory for AudioTrack size=131296
11-10 01:51:57.951  1340  1400 D MemoryDealer:   AudioTrack (0xab87d290, size=4194304)
11-10 01:51:57.951  1340  1400 D MemoryDealer:     0: 0xab87d2a0 | 0x00000000 | 0x000200E0 | A
11-10 01:51:57.951  1340  1400 D MemoryDealer:     1: 0xab87d2c0 | 0x000200E0 | 0x000200E0 | A
11-10 01:51:57.951  1340  1400 D MemoryDealer:     2: 0xab87d0c0 | 0x000401C0 | 0x000200E0 | A
11-10 01:51:57.951  1340  1400 D MemoryDealer:     3: 0xab8964e0 | 0x000602A0 | 0x000200E0 | A
11-10 01:51:57.951  1340  1400 D MemoryDealer:     4: 0xab896680 | 0x00080380 | 0x000200E0 | A
11-10 01:51:57.951  1340  1400 D MemoryDealer:     5: 0xab87d5c0 | 0x000A0460 | 0x000200E0 | A
11-10 01:51:57.951  1340  1400 D MemoryDealer:     6: 0xab87d780 | 0x000C0540 | 0x000200E0 | A
11-10 01:51:57.951  1340  1400 D MemoryDealer:     7: 0xab896970 | 0x000E0620 | 0x000200E0 | A
11-10 01:51:57.951  1340  1400 D MemoryDealer:     8: 0xab87d3d0 | 0x00100700 | 0x000200E0 | A
11-10 01:51:57.951  1340  1400 D MemoryDealer:     9: 0xab896ae0 | 0x001207E0 | 0x000200E0 | A
11-10 01:51:57.951  1340  1400 D MemoryDealer:    10: 0xab87d9d0 | 0x001408C0 | 0x000200E0 | A
11-10 01:51:57.951  1340  1400 D MemoryDealer:    11: 0xab896cb0 | 0x001609A0 | 0x000200E0 | A
11-10 01:51:57.951  1340  1400 D MemoryDealer:    12: 0xab87db50 | 0x00180A80 | 0x000200E0 | A
11-10 01:51:57.951  1340  1400 D MemoryDealer:    13: 0xab87dc00 | 0x001A0B60 | 0x000200E0 | A
11-10 01:51:57.951  1340  1400 D MemoryDealer:    14: 0xab87dcb0 | 0x001C0C40 | 0x000200E0 | A
11-10 01:51:57.951  1340  1400 D MemoryDealer:    15: 0xab87db20 | 0x001E0D20 | 0x000200E0 | A
11-10 01:51:57.951  1340  1400 D MemoryDealer:    16: 0xab87ded0 | 0x00200E00 | 0x000200E0 | A
11-10 01:51:57.951  1340  1400 D MemoryDealer:    17: 0xaac6f120 | 0x00220EE0 | 0x000200E0 | A
11-10 01:51:57.951  1340  1400 D MemoryDealer:    18: 0xa9e7f0b0 | 0x00240FC0 | 0x000200E0 | A
11-10 01:51:57.951  1340  1400 D MemoryDealer:    19: 0xa9e7f160 | 0x002610A0 | 0x000200E0 | A
11-10 01:51:57.951  1340  1400 D MemoryDealer:
11-10 01:51:57.951  1340  1400 E AudioFlinger: createRecordTrack_l() initCheck failed -12; no control block?
11-10 01:51:57.951  2056  3565 E AudioRecord: AudioFlinger could not create record track, status: -12
11-10 01:51:57.969  2056  3565 E AudioRecord-JNI: Error creating AudioRecord instance: initialization check failed with status -12.
11-10 01:51:57.969  2056  3565 E android.media.AudioRecord: Error code -20 when initializing native AudioRecord object.
11-10 01:51:57.969  2056  3565 I MicrophoneInputStream: mic_started com.google.android.apps.gsa.staticplugins.aa.c@369385
11-10 01:51:57.978  2056  3565 E ActivityThread: Failed to find provider info for com.google.android.apps.gsa.testing.ui.audio.recorded
11-10 01:51:57.978  2056  2056 I MicroDetectionWorker: onReady
11-10 01:51:57.981  2056  3565 I MicrophoneInputStream: mic_close com.google.android.apps.gsa.staticplugins.aa.c@369385
11-10 01:51:57.982  2056  2207 I MicroRecognitionRunner: Detection finished
11-10 01:51:57.982  2056  2207 W ErrorReporter: reportError [type: 211, code: 524300]: Error reading from input stream
11-10 01:51:57.987  2056  3567 I MicroRecognitionRunner: Stopping hotword detection.
11-10 01:51:58.004  2056  2207 W ErrorProcessor: onFatalError, processing error from engine(4)
11-10 01:51:58.004  2056  2207 W ErrorProcessor: com.google.android.apps.gsa.shared.speech.b.g: Error reading from input stream
11-10 01:51:58.004  2056  2207 W ErrorProcessor: 	at com.google.android.apps.gsa.staticplugins.recognizer.j.a.a(SourceFile:28)
11-10 01:51:58.004  2056  2207 W ErrorProcessor: 	at com.google.android.apps.gsa.staticplugins.recognizer.j.b.run(SourceFile:15)
11-10 01:51:58.004  2056  2207 W ErrorProcessor: 	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:457)
11-10 01:51:58.004  2056  2207 W ErrorProcessor: 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
11-10 01:51:58.004  2056  2207 W ErrorProcessor: 	at com.google.android.apps.gsa.shared.util.concurrent.a.ax.run(SourceFile:14)
11-10 01:51:58.004  2056  2207 W ErrorProcessor: 	at com.google.android.apps.gsa.shared.util.concurrent.a.bl.run(SourceFile:4)
11-10 01:51:58.004  2056  2207 W ErrorProcessor: 	at com.google.android.apps.gsa.shared.util.concurrent.a.bl.run(SourceFile:4)
11-10 01:51:58.004  2056  2207 W ErrorProcessor: 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1162)
11-10 01:51:58.004  2056  2207 W ErrorProcessor: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:636)
11-10 01:51:58.004  2056  2207 W ErrorProcessor: 	at java.lang.Thread.run(Thread.java:764)
11-10 01:51:58.004  2056  2207 W ErrorProcessor: 	at com.google.android.apps.gsa.shared.util.concurrent.a.ai.run(SourceFile:6)
11-10 01:51:58.004  2056  2207 W ErrorProcessor: Caused by: com.google.android.apps.gsa.shared.exception.GsaIOException: Error code: 393238 | Buffer overflow, no available space.
11-10 01:51:58.004  2056  2207 W ErrorProcessor: 	at com.google.android.apps.gsa.speech.audio.Tee.f(SourceFile:103)
11-10 01:51:58.004  2056  2207 W ErrorProcessor: 	at com.google.android.apps.gsa.speech.audio.au.read(SourceFile:2)
11-10 01:51:58.004  2056  2207 W ErrorProcessor: 	at java.io.InputStream.read(InputStream.java:101)
11-10 01:51:58.004  2056  2207 W ErrorProcessor: 	at com.google.android.apps.gsa.speech.audio.ao.run(SourceFile:18)
11-10 01:51:58.004  2056  2207 W ErrorProcessor: 	at com.google.android.apps.gsa.speech.audio.an.run(SourceFile:2)
11-10 01:51:58.004  2056  2207 W ErrorProcessor: 	... 9 more
11-10 01:51:58.004  2056  2207 I AudioController: internalShutdown
11-10 01:51:58.014  2056  2056 I MicroDetector: Keeping mic open: false
11-10 01:51:58.014  2056  2056 I MicroDetectionWorker: #onError(false)
11-10 01:51:58.014  2056  2219 I DeviceStateChecker: DeviceStateChecker cancelled
11-10 01:51:58.021  1508  3863 I OpenGLRenderer: Initialized EGL, version 1.4
11-10 01:51:58.021  1508  3863 D OpenGLRenderer: Swap behavior 1
11-10 01:51:58.025  1508  3863 W OpenGLRenderer: Failed to choose config with EGL_SWAP_BEHAVIOR_PRESERVED, retrying without...
11-10 01:51:58.025  1508  3863 D OpenGLRenderer: Swap behavior 0
11-10 01:51:58.036  1508  3863 D EGL_emulation: eglCreateContext: 0x8b366860: maj 2 min 0 rcv 2
11-10 01:51:58.046  1508  3863 D EGL_emulation: eglMakeCurrent: 0x8b366860: ver 2 0 (tinfo 0xa1353500)
11-10 01:51:58.079  1508  3863 D         : SurfaceInterface::setAsyncMode: set async mode 1
11-10 01:51:58.090  1508  3863 D EGL_emulation: eglMakeCurrent: 0x8b366860: ver 2 0 (tinfo 0xa1353500)
11-10 01:51:58.340  1508  1524 W Looper  : Dispatch took 423ms on android.ui, h=Handler (android.view.Choreographer$FrameHandler) {3ad151} cb=android.view.Choreographer$FrameDisplayEventReceiver@a77fdb6 msg=0
11-10 01:51:58.353  2103  2217 W OpenGLRenderer: Incorrectly called buildLayer on View: ShortcutAndWidgetContainer, destroying layer...
</pre></details>

I'm gonna venture that it has something to do with:

<details>
<summary>Specific <code>AndroidRuntime</code> crash trace.</summary>
<pre>
11-10 01:51:57.539  5114  5134 E AndroidRuntime: FATAL EXCEPTION: Thread-2
11-10 01:51:57.539  5114  5134 E AndroidRuntime: Process: com.allaboutolaf, PID: 5114
11-10 01:51:57.539  5114  5134 E AndroidRuntime: java.lang.NoClassDefFoundError: Failed resolution of: Lcom/google/android/gms/maps/GoogleMapOptions;
11-10 01:51:57.539  5114  5134 E AndroidRuntime: 	at com.airbnb.android.react.maps.AirMapManager.(AirMapManager.java:52)
11-10 01:51:57.539  5114  5134 E AndroidRuntime: 	at com.airbnb.android.react.maps.MapsPackage.createViewManagers(MapsPackage.java:39)
11-10 01:51:57.539  5114  5134 E AndroidRuntime: 	at com.facebook.react.ReactInstanceManager.createAllViewManagers(ReactInstanceManager.java:655)
11-10 01:51:57.539  5114  5134 E AndroidRuntime: 	at com.facebook.react.CoreModulesPackage.createUIManager(CoreModulesPackage.java:227)
11-10 01:51:57.539  5114  5134 E AndroidRuntime: 	at com.facebook.react.CoreModulesPackage.access$200(CoreModulesPackage.java:76)
11-10 01:51:57.539  5114  5134 E AndroidRuntime: 	at com.facebook.react.CoreModulesPackage$8.get(CoreModulesPackage.java:153)
11-10 01:51:57.539  5114  5134 E AndroidRuntime: 	at com.facebook.react.CoreModulesPackage$8.get(CoreModulesPackage.java:150)
11-10 01:51:57.539  5114  5134 E AndroidRuntime: 	at com.facebook.react.LazyReactPackage.createNativeModules(LazyReactPackage.java:82)
11-10 01:51:57.539  5114  5134 E AndroidRuntime: 	at com.facebook.react.NativeModuleRegistryBuilder.processPackage(NativeModuleRegistryBuilder.java:106)
11-10 01:51:57.539  5114  5134 E AndroidRuntime: 	at com.facebook.react.ReactInstanceManager.processPackage(ReactInstanceManager.java:1009)
11-10 01:51:57.539  5114  5134 E AndroidRuntime: 	at com.facebook.react.ReactInstanceManager.createReactContext(ReactInstanceManager.java:933)
11-10 01:51:57.539  5114  5134 E AndroidRuntime: 	at com.facebook.react.ReactInstanceManager.access$600(ReactInstanceManager.java:109)
11-10 01:51:57.539  5114  5134 E AndroidRuntime: 	at com.facebook.react.ReactInstanceManager$4.run(ReactInstanceManager.java:746)
11-10 01:51:57.539  5114  5134 E AndroidRuntime: 	at java.lang.Thread.run(Thread.java:764)
11-10 01:51:57.539  5114  5134 E AndroidRuntime: Caused by: java.lang.ClassNotFoundException: com.google.android.gms.maps.GoogleMapOptions
11-10 01:51:57.539  5114  5134 E AndroidRuntime: 	at java.lang.VMClassLoader.findLoadedClass(Native Method)
11-10 01:51:57.539  5114  5134 E AndroidRuntime: 	at java.lang.ClassLoader.findLoadedClass(ClassLoader.java:738)
11-10 01:51:57.539  5114  5134 E AndroidRuntime: 	at java.lang.ClassLoader.loadClass(ClassLoader.java:363)
11-10 01:51:57.539  5114  5134 E AndroidRuntime: 	at java.lang.ClassLoader.loadClass(ClassLoader.java:312)
11-10 01:51:57.539  5114  5134 E AndroidRuntime: 	... 14 more
11-10 01:51:57.539  5114  5134 E AndroidRuntime: Caused by: java.lang.NoClassDefFoundError: Failed resolution of: Lcom/google/android/gms/common/internal/safeparcel/zza;
11-10 01:51:57.539  5114  5134 E AndroidRuntime: 	... 14 more
11-10 01:51:57.539  5114  5134 E AndroidRuntime: Caused by: java.lang.ClassNotFoundException: Didn't find class "com.google.android.gms.common.internal.safeparcel.zza" on path: DexPathList[[zip file "/data/app/com.allaboutolaf-w2SIV_V0DGS4CaY36VQmFA==/base.apk"],nativeLibraryDirectories=[/data/app/com.allaboutolaf-w2SIV_V0DGS4CaY36VQmFA==/lib/x86, /system/fake-libs, /data/app/com.allaboutolaf-w2SIV_V0DGS4CaY36VQmFA==/base.apk!/lib/x86, /system/lib, /system/vendor/lib]]
11-10 01:51:57.539  5114  5134 E AndroidRuntime: 	at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:93)
11-10 01:51:57.539  5114  5134 E AndroidRuntime: 	at java.lang.ClassLoader.loadClass(ClassLoader.java:379)
11-10 01:51:57.539  5114  5134 E AndroidRuntime: 	at java.lang.ClassLoader.loadClass(ClassLoader.java:312)
11-10 01:51:57.539  5114  5134 E AndroidRuntime: 	... 14 more
</pre></details>

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -30,9 +30,5 @@ allprojects {
             // react-native-custom-tabs requires this maven repository
             url "https://jitpack.io"
         }
-        maven {
-            // currently used for bugsnag-react-native
-            url 'https://maven.google.com'
-        }
     }
 }

--- a/package.json
+++ b/package.json
@@ -53,7 +53,7 @@
     "@hawkrives/react-native-search-bar": "3.0.0-3",
     "@hawkrives/react-native-sortable-list": "1.0.1",
     "buffer": "5.0.8",
-    "bugsnag-react-native": "2.6.0",
+    "bugsnag-react-native": "2.5.1",
     "css-select": "1.2.0",
     "dedent": "0.7.0",
     "delay": "2.0.0",

